alerta/app.py

```diff
@@ -33,13 +34,15 @@
 
 db = Database()
 qb = QueryBuilder()
-sentry = Sentry()
+# Sentry will grab DSN from SENTRY_DSN environment variable.
+sentry_sdk.init(None, integrations=[FlaskIntegration()])
```

Comment:
Can you remove this `None`?

Reply:
Yes. Sorry about that, for some reason I thought this was a positional argument